web/src/app/listing-detail/component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from '@angular/core';
+import {Location} from '@angular/common';
+import { Router, ActivatedRoute, Params } from '@angular/router';
+import { Term, Course, Listing } from 'yacs-api-client';
+import { YacsService } from '../services/yacs.service';
```

Comment:
I don't think this is needed

Reply:
Do you mean for all the imports? I use Location, ActivatedRoute, Params, and Listing but I can get rid of the other ones.

Reply:
Talking about `YacsService`